### PR TITLE
Fix TARGET for BCI tumblweed images

### PR DESCRIPTION
**Pull request by mloviska**

### CODE REVIEW

File: bci_tester/data.py [hunk 1]
```diff
@@ -161,6 +162,8 @@ def _get_repository_name(image_type: _IMAGE_TYPE_T) -> str:
         return ""
     if TARGET == "ibs-cr":
         return "images/"
+    if TARGET == "factory-totest":
+        return "containers/opensuse/bci/"
```
Comment (dcermak) [inline bci_tester/data.py:166]: Are you sure about this? This only makes sense if all images are released into the `containers` namespace in ToTest.

Comment (dirkmueller) [inline bci_tester/data.py:166]:
```suggestion
        return "containers/opensuse/"
```

**Review by dirkmueller — CHANGES_REQUESTED** [review]
no, this doesn't work... the /bci/ suffix is duplicated

**Review by dirkmueller — APPROVED** [review]

**Comment by dirkmueller** [comment]
tested with `OS_VERSION=tumbleweed TARGET=factory-totest` :+1:

File: bci_tester/data.py (hunk 1 above)
Comment (mloviska) [reply bci_tester/data.py]: AFAICT this applies for `init, minimal, micro, busybox, golang, rust, ruby and postgres14` images that I have checked.

**Comment by mloviska** [comment]
Thank you @dirkmueller, @dcermak !

**Pull request merged**